paasta_tools/cli/cmds/status.py

```diff
@@ -1049,6 +1082,51 @@ def print_tron_status(
     return 0
 
 
+def print_kafka_status(
```

Comment:
We should maybe add a test for this? See TestPrintKubernetesStatus for an example